metals/src/main/scala/scala/meta/internal/metals/doctor/ProblemResolver.scala

```diff
@@ -176,16 +176,30 @@ class ProblemResolver(
       futureMessage,
     ).flatten
 
+    def noSemanticDBMessages = {
```

Comment:
Could we instead create bazel message instead of the basic semanticdb ones if the current build server is bazel?

Reply:
If I understand correctly, we do (see `Messages.CheckDoctor.bazelNavigation`)

Reply:
What I mean is that `noSemanticDBMessages` is used to figure out if there are no semanticdb messages, why do we need this as a fllback, instead of checking for semanticdb flags together with bazel?

Reply:
Ah right, I will try to change that